lib/brpoplpush/redis_script/config.rb

```diff
@@ -2,35 +2,49 @@
 
 module Brpoplpush
   module RedisScript
-    # ThreadSafe config exists to be able to document the config class without errors
-    ThreadSafeConfig = Concurrent::MutableStruct.new("ThreadSafeConfig",
-                                                     :debug_lua,
-                                                     :logger,
-                                                     :script_directory)
-
     #
-    # Shared class for dealing with gem configuration
+    # Class holding gem configuration
     #
-    # @author Mauro Berlanda <mauro.berlanda@gmail.com>
-    class Config < ThreadSafeConfig
+    # @author Mikael Henriksson <mikael@mhenrixon.com>
+    class Config
```

Comment:
It may be worth to add some dumb tests to make sure nobody will mess up with this class in future